torch/_inductor/codegen/wrapper.py

```diff
@@ -1286,6 +1326,31 @@ def g(args):
             """
         )
 
+    def generate_c_shim_extern_kernel_call(self, kernel, args):
```

Comment:
It would be good to have some sort of assert making sure you don't try to call kernels that aren't actually in the shim